pbs/pbsrequest.go

```diff
@@ -81,10 +87,12 @@ type PBSRequest struct {
 	IsDebug       bool            `json:"is_debug"`
 	App           *openrtb.App    `json:"app"`
 	Device        *openrtb.Device `json:"device"`
-	User          *openrtb.User   `json:"user"`
+	PBSUser       json.RawMessage `json:"user"`
+	SDK           *SDK            `json:"sdk"`
```

Comment:
`sdk` is a field specific to Prebid Mobile and is not an OpenRTB field. I didn't add it to the static pbs_request.json for that reason. Its for internal use to pass Android/iOS and version in case we need to support certain behaviors, like this backwards compatibility case.